src/patients/appointments/AppointmentsList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useHistory } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { TextInput, Button, List, ListItem, Container, Row } from '@hospitalrun/components'
+import { RootState } from '../../store'
+import { fetchPatientAppointments } from '../../scheduling/appointments/appointments-slice'
+
+interface Props {
+  patientId: string
+}
+
+const AppointmentsList = (props: Props) => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const { t } = useTranslation()
+
+  const { patientId } = props
+  const { appointments } = useSelector((state: RootState) => state.appointments)
+  const [searchText, setSearchText] = useState<string>('')
+
+  useEffect(() => {
+    dispatch(fetchPatientAppointments(patientId))
+  }, [dispatch, patientId])
+
+  const list = (
+    // inline style added to pick up on newlines for string literal
+    <ul style={{ whiteSpace: 'pre-line' }}>
+      {appointments.map((a) => (
+        <ListItem action key={a.id} onClick={() => history.push(`/appointments/${a.id}`)}>
+          {`${t('scheduling.appointment.location')}: ${a.location}
+          ${t('scheduling.appointment.reason')}: ${a.reason}
+          ${t('scheduling.appointment.type')}: ${a.type}
+          ${t('scheduling.appointment.startDate')}: ${new Date(a.startDateTime).toLocaleString()}
+          ${t('scheduling.appointment.endDate')}: ${new Date(a.endDateTime).toLocaleString()}`}
+        </ListItem>
+      ))}
+    </ul>
+  )
+
+  const onSearchBoxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchText(event.target.value)
+  }
+
+  const onSearchFormSubmit = (event: React.FormEvent | React.MouseEvent) => {
+    event.preventDefault()
+    dispatch(fetchPatientAppointments(patientId, searchText))
+  }
+
+  return (
+    <Container>
+      <form className="form-inline" onSubmit={onSearchFormSubmit}>
+        <div className="input-group" style={{ width: '100%' }}>
+          <TextInput
+            size="lg"
+            value={searchText}
+            placeholder={t('actions.search')}
+            onChange={onSearchBoxChange}
+          />
+          <div className="input-group-append">
+            <Button onClick={onSearchFormSubmit}>{t('actions.search')}</Button>
+          </div>
```

Comment:
Now that we have two search boxes, we should consider abstracting that to a component in the `/components` folder.